chapters/en/slides/module8/module8_02.md

```diff
@@ -0,0 +1,335 @@
+
+You can create a sample book by running the following command:
+
+```
+jupyter-book create mynewbook/
+```
+
+This will generate a mini Jupyter Book in your current working directory that you can both build and explore locally.
+
+The created directory `mynewbook/` should look like this:
+
+```
+mynewbook/
+├── _config.yml
+├── _toc.yml
+├── intro.md
+├── logo.png
+├── markdown-notebooks.md
+├── markdown.md
+├── notebooks.ipynb
+├── references.bib
+└── requirements.txt
+```
+
+It will have a few decisions made for you, and you can explore the configuration of the book in `_config.yml` and its structure in `_toc.yml`.
+
+We will use this sample Jupyter Book as a starting point to work from.
+
+---
+
+# Anatomy of a Jupyter Book
+
+<br>
+
+There are three things that you need in order to build a Jupyter Book, each of which was just created by running `jupyter-book create`:
+
+- A configuration file (`_config.yml`)
+
+- A table of contents file (`_toc.yml`)
+
+- Your book’s content
+
+Notes:
+
+There are three things that you need in order to build a Jupyter Book, each of which was just created by running `jupyter-book create`:
+
+- A configuration file (`_config.yml`)
+
+- A table of contents file (`_toc.yml`)
+
+- Your book’s content
+
+<br>
+
+There are a few extra files in there included to demonstrate some features, but the required pieces are `_toc.yml`, `_config.yml`, and content files.
+
+We’ll cover each briefly in the next slides.
+
+---
+
+# Book configuration (`_config.yml`)
+
+<br>
+
+All of the configuration for your book is in a YAML file called `_config.yml`.
+
+<br>
+
+Here’s an example of a simple `_config.yml` file:
+
+```yaml
+# In _config.yml
+title: My sample book
+author: The Jupyter Book Community
+logo: logo.png
+execute:
+  execute_notebooks: force
+
+# Add a bibtex file so that we can create citations
+bibtex_bibfiles:
+  - references.bib
+```
+
+Notes:
+
+All of the configuration for your book is in a YAML file called `_config.yml`.
+
+You can define metadata for your book (such as its title), add a book logo, turn on different “interactive” buttons (such as a Binder button for pages built from a Jupyter Notebook), and more.
+
+Here’s an example of a simple `_config.yml` file:
+
+- `title`: defines a title for the book. It will show up in the left sidebar.
+
+- `author`: adds the author’s name to your book template, for attribution.
+
+- `logo`: defines a path to an image file for your book’s logo (it will also show up in the sidebar).
+
+- `execute`: contains a collection of configuration options to control execution and cacheing.
+
+- `execute_notebooks`: "force" tells Jupyter Book force execute any computational content each time it builds the book. By default, Jupyter Book executes and caches all book content.
+
+- `bibtex_bibfiles`: is a section to define bibliography files for your Jupyter Book. This configuration activates citations for your book.
+
+---
+
+# Table of contents (`_toc.yml`)
```

Comment:
Could we include parts https://jupyterbook.org/en/stable/structure/toc.html#use-parts-to-organize-chapters in a later slide?

Reply:
Maybe also a mention that the chart title will be used in the navigation side bar? https://jupyterbook.org/en/stable/structure/sections-headers.html